buildSrc/src/main/java/com/uber/okbuck/composer/android/ManifestRuleComposer.java

```diff
@@ -19,13 +20,15 @@ public static Rule composeForBinary(AndroidTarget target) {
   }
 
   private static ManifestRule compose(AndroidTarget target) {
-    return new ManifestRule()
-        .debuggable(target.getDebuggable())
-        .minSdk(target.getMinSdk())
-        .targetSdk(target.getTargetSdk())
-        .versionCode(target.getVersionCode())
-        .versionName(target.getVersionName())
-        .mainManifest(target.getMainManifest())
-        .secondaryManifests(target.getSecondaryManifests());
+    return (ManifestRule)
```

Comment:
can probably just return `Rule`

Reply:
Can not do, this is private method used by 2 public methods few lines above.